gallery/src/styles/NxCard/NxCardHorizontalLayoutExample.tsx

```diff
@@ -8,93 +8,87 @@ import React from 'react';
 import { faShapes, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
 
 import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+const chart = require('./chart.png');
 
 export default function NxCardColumnLayoutExample() {
   return (
     <div className="nx-card-container nx-card-container--column">
-      <section className="nx-card" aria-label="card 1 in column layout">
+      <section className="nx-card nx-card--horizontal" aria-label="card 1 in column layout">
```

Comment:
Wasn't the idea previously that `.nx-card--horizontal` should be redundant because of `.nx-card-container--column`?

Reply:
> Wasn't the idea previously that `.nx-card--horizontal` should be redundant because of `.nx-card-container--column`?

I think that making the card's appearance/layout dependent on their container was a mistake. That's why I moved over to a modifier class on `nx-card`